backend/src/api/candidates.js

```diff
@@ -296,16 +296,16 @@ router.put(
 )
 router.delete(
   '/:candidateId/interviews/:interviewId',
-  [leadsOnly],
+  [directorsOnly],
   errorWrap(async (req, res) => {
     let response = 'Interview Deleted Sucessfully'
```

Comment:
hmmm unless I am missing some context here, the response never changes. Should it be changed to a `const` instead?

Reply:
fixed, will update other ones also